# HG changeset patch
# User Philip Nienhuis <prnienhuis@users.sf.net>
# Date 1373575622 -7200
#      Thu Jul 11 22:47:02 2013 +0200
# Node ID db4453f405503a98a83caeaaf610a47d03c4af44
# Parent  7c575556e8175ad1292e2157193ba3fdc1c73162
configure.ac: fix  building Java support for MinGW

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2522,16 +2522,17 @@ do
     ;;
     mingw* | cygwin*)
       if test -n "$JAVA_CPPFLAGS"; then
         JAVA_CPPFLAGS="-I${JAVA_CPPFLAGS}/include -I${JAVA_CPPFLAGS}/include/win32"
       else
         JAVA_CPPFLAGS="-I${JAVA_HOME}/include -I${JAVA_HOME}/include/win32"
       fi
       JAVA_LIBS=-ladvapi32
+      LDFLAGS="$LDFLAGS -Wl,--export-all-symbols"
     ;;
     *)
       if test -n "$JAVA_CPPFLAGS"; then
         JAVA_CPPFLAGS="-I${JAVA_CPPFLAGS}/include -I${JAVA_CPPFLAGS}/include/linux"
       else
         JAVA_CPPFLAGS="-I${JAVA_HOME}/include -I${JAVA_HOME}/include/linux"
       fi
     ;;
